woocommerce/woocommerce
# Create editor skeleton on add/edit product pages

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/36991



### How to test the changes in this Pull Request:




1. Visit the urls `/wp-admin/admin.php?page=wc-admin&path=/add-product` and `http://woodev.local/wp-admin/admin.php?page=wc-admin&path=/product/{productId}`.
2. You should see the GB block editor and interact with it.



### Other information:




### FOR PR REVIEWER ONLY:
